dev/bots/test.dart

```diff
@@ -1602,6 +1603,41 @@ Future<void> _runAnalyze() async {
   );
 }
 
+// Runs flutter_precache.
+Future<void> _runFuchsiaPrecache() async {
+  printProgress('${green}Running flutter precache tests$reset');
+  await runCommand(
+    'flutter',
+    <String>[
+      'config',
+      '--enable-fuchsia',
+    ],
+    workingDirectory: flutterRoot,
+  );
+  await runCommand(
+    'flutter',
+    <String>[
+      'precache',
+      '--fuchsia',
+      '--no-android',
+      '--no-ios',
+      '--force',
+    ],
+    workingDirectory: flutterRoot,
+  );
+  await runCommand(
+    'flutter',
+    <String>[
+      'precache',
+      '--flutter_runner',
```

Comment:
could this just be combined with the previous `flutter precache` command?

Reply:
Yes, I don't see why not (unless there's some hidden gotcha in the flutter precache command I'm not aware of, in which case we should just fix it)

Reply:
ok, sending an update.

Reply:
Led run with the new changes: https://chromium-swarm.appspot.com/task?id=6635f944ed3c5810